deltachat-ffi/src/lib.rs

```diff
@@ -618,6 +620,10 @@ pub unsafe extern "C" fn dc_event_get_data2_int(event: *mut dc_event_t) -> libc:
             status_update_serial,
             ..
         } => status_update_serial.to_u32() as libc::c_int,
+        EventType::WebxdcUpdateStateChanged {
+            is_sending: is_send,
```

Comment:
why using two wordings here? `is_send` also look grammatically wrong to me - and `is_sent` is probably not what is meant here.